docker/package/model.py

```diff
@@ -584,9 +585,16 @@ def gen_makefile(self, out):
         file_contents = """
 .PHONY: install
 
+BINDIR=/usr/bin
+
 tezos-baking:
 
-install: tezos-baking
+tezos-baking-wizard:
+	chmod +x $(DESTDIR)$(BINDIR)/tezos-baking-wizard
```

Comment:
hmm, this doesn't look right here :thinking:
should be something like: 
```suggestion
        mv $(CURDIR)/tezos_baking_wizard.py $(CURDIR)/tezos-baking-wizard
	chmod +x $(CURDIR)/tezos-baking-wizard
```